Models/DataAccess/Company.php

```diff
@@ -349,18 +366,27 @@ public function update(CompanyData $company, $id)
 		$query->bindValue(11, isset($company->visibleTags[1]) ? $company->visibleTags[1] : '', PDO::PARAM_STR);
 		$query->bindValue(12, $otherTags, PDO::PARAM_STR);
 		$query->bindValue(13, time(), PDO::PARAM_INT);
-		$query->bindValue(14, $id, PDO::PARAM_INT);
+
+		$query->bindValue(14,$company->postalCode, PDO::PARAM_STR);
+
+		$query->bindValue(15, $id, PDO::PARAM_INT);
 		$query->execute();
 
 		$socialMedia = new SocialMedia($this->pdo);
 		$socialMedia->update($company->instagram, $id);
 	}
 
+    public function fetchProvinceID($postalCode){
+        return $this->fetchQuery("select Province.ProvinceID from Province inner join PostalCode on PostalCode.ProvinceID = Province.ProvinceID where PostalCode.Code ='".$postalCode."'");
```

Comment:
This line is way too long. Try to keep your code line shorter than 120 char please 🥺
And an SQL injection is possible here with the variable $postalCode I think...

Reply:
Noted. I've resolved this in the latest commit.